plugins/node/opentelemetry-instrumentation-aws-lambda/src/types.ts

```diff
@@ -40,4 +40,5 @@ export interface AwsLambdaInstrumentationConfig extends InstrumentationConfig {
   responseHook?: ResponseHook;
   disableAwsContextPropagation?: boolean;
   eventContextExtractor?: EventContextExtractor;
+  handler?: string;
```

Comment:
The name `handler` is a bit general.
WDYT about calling it `lambdaHandler` so it is clearer the value is related to the lambda config settings and not an instrumentation option.

Reply:
Yes, I agree. I've made this update ✅